packages/mocha/README.md

```diff
@@ -68,10 +68,13 @@ Now update your project's `.neutrinorc.js` to add the `@neutrinojs/mocha` preset
 let's pretend this is a Node.js project:
 
 ```js
+const node = require('@neutrinojs/node');
+const mocha = require('@neutrinojs/mocha');
+
 module.exports = {
   use: [
-    '@neutrinojs/node',
-    '@neutrinojs/mocha'
+    node(),
+    mocha()
   ]
 };
 ```
```

Comment:
The "Preset options" section just below here needs updating to migrate the `['@neutrinojs/mocha', { bail: true }]` and also the "Use an array pair instead of a string" sentence.